test/dotnet-test.UnitTests/GivenAUnknownMessageHandler.cs

```diff
@@ -10,11 +10,15 @@
 
 namespace Microsoft.Dotnet.Tools.Test.Tests
 {
-    public class GivenAUnknownMessageHandler
+    public class GivenAnUnknownMessageHandler
     {
         [Fact]
         public void It_throws_InvalidOperationException_and_sends_an_error_when_the_message_is_not_handled()
         {
+            // We need to re-think the tracing story. For the time being, enabling this hack to get around
```

Comment:
Github issue if we are ever really going to fix this. Otherwise, just remove the comment. Be realistic on whether we will really fix this or not.